features/cellular/framework/AT/AT_CellularContext.cpp

```diff
@@ -319,6 +319,7 @@ void AT_CellularContext::set_credentials(const char *apn, const char *uname, con
     _apn = apn;
     _uname = uname;
     _pwd = pwd;
+    tr_info("set_credentials APN %s USERNAME %s PASSWORD %s", _apn, _uname, _pwd);
```

Comment:
Tracing credentials is not allowed. Please remove.

Reply:
done